src/v3/src/util/theme.ts

```diff
@@ -307,6 +314,22 @@ export const createThemeAndTokens = (
           },
         },
       },
+      MuiScopedCssBaseline: {
+        styleOverrides: {
+          root: {
+            figure: {},
+          },
+        },
+      },
+      MuiTypography: {
+        styleOverrides: {
+          root: {
+            '&[data-se="o-form-head"]': {
+              outline: 'none',
```

Comment:
Adds back `outline: 'none'` style for `Title.tsx` component at the theme level so we can still use Odyssey `Typography`